app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# The class representing an actual user
+class User < OrganizationalUnit
+  plugin :devise
+
+  devise :database_authenticatable
+
+  def personal_repositories
+    Repository.
```

Comment:
Could be written as just `namespace.repositories`, but this only uses one query instead of two.